libnm/libnm.ver

```diff
@@ -948,6 +949,7 @@ global:
 	nm_setting_connection_autoconnect_slaves_get_type;
 	nm_setting_connection_get_autoconnect_slaves;
 	nm_setting_connection_get_lldp;
+	nm_setting_connection_get_mdns;
```

Comment:
As 1.10-rc1 is already released, we better don't change API anymore (not even extend)

These new entries, must move to a new linker section libnm_1_12_0